build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = 'com'
+            artifactId = 'builders'
+            from components.java
+        }
+    }
+    repositories {
+        maven {
+            url = "$rootProject.buildDir/m2repo"
+        }
+    }
+}
```

Comment:
It looks like this block should be removed to avoid builders artifact publishing to maven local repository
```
publishing {
   ....
}
```